### PR TITLE
[DSD-5554] : Updated XSD Storage URI to point to collab mosip-config

### PR DESCRIPTION
**Pull request by vijay151096**

### CODE REVIEW

**Review by vharsh — APPROVED** [review]

File: mimoto-default.properties [hunk 1]
```diff
@@ -131,7 +131,7 @@ mosip.mandatory-languages=eng,khm
 
 #----------------------- CBEFF Util--------------------------------------------------
 # Cbeff URL where the files will be stored in git, change it accordingly in case of change of storage location.
-mosip.kernel.xsdstorage-uri=${spring.cloud.config.uri}/print/${spring.profiles.active}/${spring.cloud.config.label}/
+mosip.kernel.xsdstorage-uri=https://raw.githubusercontent.com/mosip/mosip-config/collab1/
```
Comment (vharsh) [inline mimoto-default.properties:134]:
```suggestion
mosip.kernel.xsdstorage-uri=https://raw.githubusercontent.com/mosip/mosip-config/develop/
```

Comment (vharsh) [inline mimoto-default.properties:134]:
```suggestion
mosip.kernel.xsdstorage-uri=https://raw.githubusercontent.com/mosip/mosip-config/develop/
```

Comment (vharsh) [inline mimoto-default.properties:134]:
```suggestion
mosip.kernel.xsdstorage-uri=https://raw.githubusercontent.com/mosip/mosip-config/develop/
```

Comment (swatigoel) [reply mimoto-default.properties:134]: No, we want to refer this based on spring profile

**Pull request closed without merging**